toqito/states/max_entangled.py

```diff
@@ -55,7 +55,7 @@ def max_entangled(dim: int, is_sparse: bool = False, is_normalized: bool = True)
     :return: The maximally entangled state of dimension :code:`dim`.
 
     """
-    mat = sp.sparse.identity(dim) if is_sparse else np.identity(dim)
+    mat = eye_array(dim) if is_sparse else np.identity(dim)
     psi = np.reshape(mat, (dim**2, 1))
```

Comment:
I can't edit the line as you haven't made changes to it directly. 

There's a typo at L53. '`True` if vector is spare' should be changed to '`True` if vector is sparse`